docs/api/flink/Function.md

```diff
@@ -1557,6 +1557,26 @@ Output:
 POLYGON ((7 -1, 7 6, 9 6, 9 1, 7 -1), (6 2, 8 2, 8 1, 6 1, 6 2))
 ```
 
+## ST_Polygon
```

Comment:
This doc is not sorted alphabeticaly

Reply:
Fixed it. I put it near st_makepolygon for that they're similar.